classes/usage/quota_manager.php

```diff
@@ -73,12 +73,17 @@ public static function get_activeusers_and_quota() {
         return $a;
     }
 
+
+    // P.T. 29-09-2022, deactivated automatic blockage of a moodle-site and activation of maintenance-mode when disk-quota exceeded
+
```

Comment:
this can be removed as well, but it would be nice to describe in english what the function does, as it only blocks if hardlimit and blocksite. Maybe with this format:
```    
    /**
     * Send notification that the site is over quoata, if a notification has not recently been sent.
     *
     * @param $used
     * @param $settings
     */
```